File: spologp_v/listareforço/questao9.c
#include<stdio.h>

int main(){
    char opcao;

switch(opcao)
    {
        case 1:
        printf("Depósito");
        break;

        case 2:
        printf("Saque");
        break;

        case 3:
        printf("Extrato");
        break;

        case 4:
        printf("Transferência");
        break;

        case E:
        printf("Encerrando o programa...");
        break;

        default:
        printf("Opção indisponível");
}

    return 0;
}

File: spologp_v/listareforço/questao9.py
opcao = str(input('"Digite uma das seguintes opções:\n 1. Depósito\n 2. Saque\n 3. Extrato\n 4. Transferência\n ou E para encerrar o programa: '))

if(opcao=="1"):
    print("Depósito")
elif(opcao=="2"):
    print("Saque")
elif(opcao=="3"):
    print("Extrato")
elif(opcao=="4"):
    print("Transferência")
elif(opcao=="E"):
    print("Encerrando o programa...")
else:
    print("Opção indisponível")